libraries/hatch-server-user-management/src/AWSCognitoClient.ts

```diff
@@ -1,21 +1,45 @@
-import UserServiceClient, {UserInfo} from './UserServiceClient';
-import {
-  AWS_ACCESS_KEY_ID,
-  AWS_CLIENT_ID,
-  AWS_USER_POOL_ID,
-  AWS_REGION,
-  AWS_SECRET_ACCESS_KEY
-} from './constants';
-import {config, CognitoIdentityServiceProvider} from 'aws-sdk';
+import {AWS_ACCESS_KEY_ID, AWS_CLIENT_ID, AWS_REGION, AWS_SECRET_ACCESS_KEY, AWS_USER_POOL_ID} from './constants';
```

Comment:
Can you update this to use a [credentials provider chain](https://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/CredentialProviderChain.html) like we do elsewhere, rather than rely on environment variables directly?

Reply:
Per offline discussion, we will inject and have a subsequent pull request to use the default provider